package.json

```diff
@@ -41,7 +41,7 @@
     "eslint-plugin-react": "^7.19.0",
     "eslint-plugin-react-hooks": "^2.5.0",
     "husky": "^3.1.0",
-    "jest": "^25.1.0",
+    "jest": "^24.9.0",
```

Comment:
downgraded?

Reply:
do we need jest at the root level?

Reply:
Downgrade is to make it sync with CRA. CRA does not support 25.

Reply:
Yes. Good question do we need jest at root? We can remove it